### PR TITLE
Dynamic array

### PR DESCRIPTION
**Pull request by YuriiHaidamaka**
In this PR we have created simple Polymer 2.0 application with one web component. 
Which has an input ("Name") and a button ("Add"). 
When user clicks the button a new line of text appears ("Hello [[name]]") where name is a value of input.

### CODE REVIEW

File: src/dynamic-array-app/dynamic-array-app.html [hunk 1]
```diff
@@ -1,32 +1,58 @@
...
+
+            handleClick() {
+                this.addName(this.$.name.value);
+            }
+
+            addName(name) {
+                this.push('names', name);
+            }
+
+            static get properties() {
```
Comment (dkalinin) [inline src/dynamic-array-app/dynamic-array-app.html]: `properties` should be declared after `is` getter.

File: src/dynamic-array-app/dynamic-array-app.html [hunk 2]
```diff
@@ -1,32 +1,58 @@
...
+         * @customElement
+         * @polymer
+         */
+        class DynamicArrayApp extends Polymer.Element {
+            static get is() {
+                return 'dynamic-array-app';
+            }
+
+            handleClick() {
+                this.addName(this.$.name.value);
```
Comment (dkalinin) [inline src/dynamic-array-app/dynamic-array-app.html]: This is good. Let's also practice data binding and get value of input from property.

File: src/dynamic-array-app/dynamic-array-app.html [hunk 3]
```diff
@@ -1,32 +1,58 @@
...
+            }
+
+            addName(name) {
+                this.push('names', name);
+            }
+
+            static get properties() {
+                return {
+                    prop1: {
```
Comment (dkalinin) [inline src/dynamic-array-app/dynamic-array-app.html]: This property is redundant.

**Comment by YuriiHaidamaka** [comment]
@dkalinin ,  PTAL

**Pull request merged**